apps/sns.py

```diff
@@ -22,9 +22,9 @@ def _publish(self, message_type, **kwargs):
     def aftership_tracking_update(self, shipment, aftership_id):
         self._publish(
             SNSClient.MessageType.AFTERSHIP_UPDATE,
-            owner_id=shipment.owner_id,
-            aftership_id=aftership_id,
-            shipment_id=shipment.id
+            ownerId=shipment.owner_id,
```

Comment:
Also why the change from snake to camel case? Is the latter somehow preferred in AWS?

Reply:
JSON messages are generally formatted in camel case